tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNetCore/FireOnStartCommon.cs

```diff
@@ -40,6 +40,24 @@ namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AspNetCore;
     MaximumVersion = "7.*.*.*",
     IntegrationName = IntegrationName,
     InstrumentationCategory = InstrumentationCategory.AppSec)]
+[InstrumentMethod(
```

Comment:
wondering if after @daniel-romano-DD [PR](https://github.com/DataDog/dd-trace-dotnet/pull/4271), can just take advantage of the flags Instrumentationcategory?

Reply:
True